lib/tasks/taxonomy.rake

```diff
@@ -0,0 +1,20 @@
+namespace :taxonomy do
+  desc <<-END_DESC
+	This task will update organization & location records with all users for which ignore_types includes 'User' resource.
+  END_DESC
+  task :update_taxonomy => :environment do
```

Comment:
the rake task should probably run in `User.anonymous_admin do ... end` (required since recent changes, probably was not required back then)

Reply:
Tried the scenario with newly created DB, this migration fails because user `ANONYMOUS_ADMIN` not being present at that time. Added condition to handle this in rake task but is there any better way to run the rake task in case this user is not present?

Reply:
The admin user should exist when this task is needed, since admins are seeded before taxonomies. If the database is not seeded at all, this task shouldn't be needed and can be skipped. The condition could check `if Organization.any?` or `if Location.any?` but it's similar to what you have already. So I guess it can remain as is.

Reply:
Okay then I am keeping the condition as it is.
Thank you @ares